Dockerfile

```diff
@@ -2,45 +2,32 @@ FROM ubuntu:18.04
 
 # Tools
 RUN apt-get update && apt-get install -y \
+    g++ \
     gdb \
     git \
-    vim \ 
+    make \
```

Comment:
How come we suddenly need to install make?

Reply:
No sure, I'll try without and check.

Reply:
For running `wtr`. We can do without but I say leave it for now.